network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/ih_cta.html

```diff
@@ -1,7 +1,5 @@
-<div class="my-5">
```

Comment:
Shouldn't need two wrapper divs here.

Reply:
We shouldn't slip that into this PR - let's leave it doubled here, and file a separate PR for "undoubling" this code

Reply:
see reasoning in https://github.com/mozilla/foundation.mozilla.org/pull/3066#discussion_r281411196